bundle/manifests/cert-manager.clusterserviceversion.yaml

```diff
@@ -616,7 +616,13 @@ spec:
           - create
         serviceAccountName: cert-manager-webhook
       deployments:
-      - name: cert-manager
+      - label:
+          app: cert-manager
+          app.kubernetes.io/component: controller
+          app.kubernetes.io/instance: cert-manager
+          app.kubernetes.io/name: cert-manager
+          app.kubernetes.io/version: v1.7.2
```

Comment:
These are added due to a bug fix in https://github.com/operator-framework/operator-sdk/releases/tag/v1.19.0

 * https://github.com/operator-framework/operator-sdk/pull/5533